lldb/source/Expression/IRMemoryMap.cpp

```diff
@@ -640,6 +640,13 @@ void IRMemoryMap::WritePointerToMemory(lldb::addr_t process_address,
                                        lldb::addr_t address, Status &error) {
   error.Clear();
 
+  /// Only ask the Process to fix the address if this address belongs to the
+  /// process (host allocations are stored in m_data).
```

Comment:
This is missing a "because...".

Even though that reason will lead most people to wonder why we fix pointers written into the process given that they could be running on the same host but ok - larger issue as we discussed before.